cmd/docs.go

```diff
@@ -37,18 +39,33 @@ var docsCmd = &cobra.Command{
 				u.LogErrorAndExit(schema.CliConfiguration{}, err)
 			}
 
+			// Detect terminal width if width flag is not set
+			width := cliConfig.Settings.Docs.MaxWidth
```

Comment:
We should probably rename this to maxWidth to better reflect it's current usage.
```suggestion
			width := cliConfig.Settings.Docs.MaxWidth
```

Reply:
Updated the naming!